internal/pkg/telemetry/metric/prometheus/prometheus.go

```diff
@@ -56,6 +57,9 @@ func ServeMetrics(ctx context.Context, serviceName, listenAddr string, logger lo
 		return nil, err
 	}
 
+	// Register legacy OpenCensus metrics, for go-cloud (https://github.com/google/go-cloud/issues/2877)
+	exporter.RegisterProducer(opencensus.NewMetricProducer())
+
```

Comment:
`go-cloud`, ktory pouzivame na pracu s S3/ABS nepodporuje zatial OpenTelemetry, viac v https://github.com/google/go-cloud/issues/2877 (postupne je tych dovodou menej, casom urcite prejdu na OpenTelemetry). Zatial je tu tento bridge.

OpenCensus je zial globalna zalezitost, preto to registrujem na najvyssej moznej urovni, aby to nerozbilo testovatelnost kodu na nizsich urovniach.